static/src/stylesheets/module/email/_form.scss

```diff
@@ -300,6 +302,26 @@
     .email-sub__text-input {
         border-color: $guardian-brand;
     }
+
+}
+
+// PLAIN-DARK MODS
+.email-sub__form--plainDark {
+    padding: 0;
+
+    .email-sub__submit-input {
+        background-color: #DDDBD7;
```

Comment:
I couldn't find a variable for this colour. Do you know if it is used elsewhere on the site @duarte ?